.circleci/config.yml

```diff
@@ -24,6 +24,7 @@ jobs:
     docker:
       - image: circleci/node:lts
     working_directory: ~/protocol
+    resource_class: large
```

Comment:
The build was running out of memory repeatedly, so I increased the docker image size.